Source/WTF/wtf/Compiler.h

```diff
@@ -641,3 +641,19 @@
 #else
 #define WTF_UNSAFE_BUFFER_USAGE
 #endif
+
+/* WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE */
+
+#define WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN \
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN \
+    ALLOW_COMMA_BEGIN \
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN \
+    IGNORE_CLANG_WARNINGS_BEGIN("cast-align") \
```

Comment:
```suggestion
    IGNORE_WARNINGS_BEGIN("cast-align") \
```

You surely want to suppress this warning when compiling with GCC as well.